assets/src/edit-story/components/panels/index.js

```diff
@@ -87,8 +90,21 @@ export const PanelTypes = {
 
 const ALL = Object.values(PanelTypes);
 
+function reorderPanels(panels) {
```

Comment:
What's the purpose of this function?

Reply:
This function is for re-ordering the panels section, otherwise, basic on the current logic, the order will be:
...
- Poster
- Playback
- Captions
- Description
(mainly due to the `intersect` function.)

So I add this function to re-order the sections. Or should we just keep the previous order? Please let me know 🙂

Reply:
Updated a comment to explain WHY.

Reply:
Actually figured this out, there is no need to add this helper function. The order logic can be controlled by the items order object `PanelTypes`. Just updated.